tests/testthat/test-methods.R

```diff
@@ -71,10 +71,7 @@ test_that("as.data.frame.lints", {
     stringsAsFactors = FALSE
   )
 
-  expect_equal(
-    df,
-    exp
-  )
+  expect_equal(df, exp)
```

Comment:
what's going on here? 🙃

Reply:
`df` columns are not of integer type here:

```r
── Failure (test-methods.R:74): as.data.frame.lints ────────────────────────────
`df` (`actual`) not identical to `exp` (`expected`).

`actual$line_number` is a double vector (1, 2)
`expected$line_number` is an integer vector (1, 2)

`actual$column_number` is a double vector (1, 6)
`expected$column_number` is an integer vector (1, 6)
```

Reply:
Was going to fix it in a different PR, but might as well do it here.

Reply:
thanks, I would have suggested just leaving the style fix for that other PR

Reply:
So should I undo this here?

Reply:
no it's fine, minor point here